scripts/test-setup.rs

```diff
@@ -0,0 +1,29 @@
+use std::env;
+use std::time::Duration;
+
+use momento::config::configurations;
+use momento::{CacheClient, CredentialProviderBuilder};
+
+#[tokio::main]
+async fn main() {
+    let cache_name =
+        env::var("TEST_CACHE_NAME").expect("environment variable TEST_CACHE_NAME should be set");
```

Comment:
any thoughts on providing a default here? it kinda always bugs me for local dev when we have to add another env var, but i can imagine some pros and cons

Reply:
That also reminds me that I forgot to make a CONTRIBUTING.md.

Reply:
discussed in slack, the current behavior seems okay, will standardize on a default fallback cache name